synapseclient/table.py

```diff
@@ -1026,21 +1026,24 @@ class Column(DictObject):
     Defines a column to be used in a table :py:class:`synapseclient.table.Schema`
     :py:class:`synapseclient.table.EntityViewSchema`.
 
-    :var id:                An immutable ID issued by the platform
-    :param columnType:      The column type determines the type of data that can be stored in a column. It can be any
-                            of: "STRING", "DOUBLE", "INTEGER", "BOOLEAN", "DATE", "FILEHANDLEID", "ENTITYID", "LINK",
-                            "LARGETEXT", "USERID". For more information, please see:
-                            https://docs.synapse.org/rest/org/sagebionetworks/repo/model/table/ColumnType.html
-    :param maximumSize:     A parameter for columnTypes with a maximum size. For example, ColumnType.STRINGs have a
-                            default maximum size of 50 characters, but can be set to a maximumSize of 1 to 1000
-                            characters.
-    :param name:            The display name of the column
-    :param enumValues:      Columns type of STRING can be constrained to an enumeration values set on this list.
-    :param defaultValue:    The default value for this column. Columns of type FILEHANDLEID and ENTITYID are not allowed
-                            to have default values.
+    :var id:                  An immutable ID issued by the platform
+    :param columnType:        The column type determines the type of data that can be stored in a column. It can be any
+                              of: "STRING", "DOUBLE", "INTEGER", "BOOLEAN", "DATE", "FILEHANDLEID", "ENTITYID", "LINK",
+                              "LARGETEXT", "USERID". For more information, please see:
+                              https://docs.synapse.org/rest/org/sagebionetworks/repo/model/table/ColumnType.html
+    :param maximumSize:       A parameter for columnTypes with a maximum size. For example, ColumnType.STRINGs have a
+                              default maximum size of 50 characters, but can be set to a maximumSize of 1 to 1000
+                              characters.
+    :param maximumListLength: Required if using a columnType with a "_LIST" suffix. Describes the maximum number of
+                              values that will appear in that list. Value range 1-100 inclusive. Default 100
```

Comment:
What happens if I set the value below the number of items in the list? Are the remaining items concatenated or dropped?

Reply:
The API throws an error if you try to set a maxlistlength that is less than the previous one you have set.   Similarly you can't set a number of values in the list greater than this limit.

Reply:
Sorry, I misunderstood the intent of this argument. I thought it was for retrieving list-columns, not specifying them in a table schema.